src/legacy/status_im/ui/components/colors.cljs

```diff
@@ -237,6 +243,6 @@
     (reset! theme-type type)))
 
 ;; Colors related to Visibility Status
-(def color-online "#7CDA00")
+(def color-online "#23ADA0")
```

Comment:
we shouldn't need to update the colors, the color should becoming from quo.foundations.colors so why did we have to make adjustments here? 🤔